ethcore/res/ethereum/transition_test.json

```diff
@@ -57,6 +57,7 @@
 		"0000000000000000000000000000000000000002": { "balance": "1", "builtin": { "name": "sha256", "pricing": { "linear": { "base": 60, "word": 12 } } } },
 		"0000000000000000000000000000000000000003": { "balance": "1", "builtin": { "name": "ripemd160", "pricing": { "linear": { "base": 600, "word": 120 } } } },
 		"0000000000000000000000000000000000000004": { "balance": "1", "builtin": { "name": "identity", "pricing": { "linear": { "base": 15, "word": 3 } } } },
+		"0000000000000000000000000000000000000005": { "builtin": { "name": "modexp", "activate_at": "0x5", "pricing": { "modexp": { "divisor": 20 } } } },
```

Comment:
Why is the `"balance"` key missing here?

Reply:
Don't know, but it seems to be the way the rest of the configs do it.
- https://raw.githubusercontent.com/paritytech/parity-ethereum/master/ethcore/res/ethereum/foundation.json
- https://github.com/paritytech/parity-ethereum/blob/master/ethcore/res/ethereum/byzantium_test.json#L56
- https://github.com/paritytech/parity-ethereum/blob/master/ethcore/res/ethereum/st_peters_test.json#L61

Reply:
The `balance` key here is to "touch" the account. Without it the account will be considered non-exist in genesis, and when the precompiled is first called, the caller pays the creation fee.

It's both okay whether with or without it.

Reply:
> It's both okay whether with or without it.

I think having it is better simply because it saves the future code reader some head scratches "why is it missing here but not in the other lines?". Not a big deal though.